src/cli/src/data/export.rs

```diff
@@ -89,6 +89,10 @@ pub struct ExportCommand {
     #[clap(long)]
     end_time: Option<String>,
 
+    /// The parallelism of the export.
+    #[clap(long, default_value = "8")]
```

Comment:
The default value should be smaller such as `4`.
We don't want to consume too much resources for exporting tasks by default.
Increase it manually if needed.